public/doc/openapi-3-v1.1.json

```diff
@@ -3663,6 +3663,35 @@
             }
           }
         }
+      },
+      "ApiError": {
+	      "type": "object",
+	      "description": "API Error",
+	      "properties": {
+		      "status": {
+			      "type": "string",
+			      "description": "HTTP status code",
+			      "example": "400"
+		      },
+		      "details": {
+			      "type": "string",
+			      "description": "Error details",
+			      "example": "Bad Request"
+		      }
+	      }
+      },
+      "ApiErrorCollection": {
+	      "type": "object",
+	      "description": "API Error collection",
+	      "properties": {
+		      "errors": {
+			      "type": "array",
+			      "description": "Error list from the API query",
+			      "items": {
+				      "$ref": "#/components/schemas/ApiError"
+			      }
+		      }
+	      }
```

Comment:
This is probably a silly ask, but can we change the indentation to match the rest of the file?